cloud/fallback-public-cloud.yaml

```diff
@@ -235,19 +251,35 @@ clouds:
         endpoint: https://us-east-2.api.joyentcloud.com
       us-east-3:
         endpoint: https://us-east-3.api.joyentcloud.com
+      us-east-3b:
```

Comment:
This is the early access region. I don't think we want to expose it TBH.

Reply:
removed